src/konserve_jdbc/core.clj

```diff
@@ -214,11 +234,26 @@
 (defn connect-store [db-spec & {:keys [table opts]
                                 :or {table default-table}
                                 :as params}]
+
+  (when-not (:debug opts)
+      (System/setProperties 
```

Comment:
Can you add a comment why this is necessary?

Reply:
Good point @whilo . I added a function to `dissoc` the closed connection from the pool.

Reply:
Now I took a deeper look into c3p0 to understand these properties and as I understand it, setting `com.mchange.v2.log.MLog` to `com.mchange.v2.log.FallbackMLog` sets c3p0 to log to stderr and then `com.mchange.v2.log.FallbackMLog.DEFAULT_CUTOFF_LEVEL` sets 'loglevel' to `OFF`. 
Does this mean that you are taking configurability for logging of c3p0 from the implementing application away? I would rather direct the implementing user to set the properties in a sane way in his/her config. Or is it possible to overwrite these properties by setting them at start?

https://www.mchange.com/projects/c3p0/#configuring_logging

Reply:
not objecting a merge, though

Reply:
@TimoKramer @whilo we're ready. I found a more elegant solution after reading the docs in that link. 

Basically we route the logs to `Slf4j` and install a `timbre` adapter (https://github.com/fzakaria/slf4j-timbre). `timbre` now controls all logging. Drama = gone.